### PR TITLE
source/packaging-guide/manual.rst:add anchor to "this repository"

### LINKED ISSUE #77: Missing link to AppRun file in manual packaging doc
https://github.com/AppImage/docs.appimage.org/blob/master/source/packaging-guide/manual.rst

> The AppRun file can be a script or executable. It sets up required environment variables such as $PATH and launches the payload application. You can write your own, but in most cases it is easiest (and most error-proof) to use a precompiled one from **this repository**.

I assume under "this repository" should be link, otherwise it's confusing.

### PR DESCRIPTION
**Pull request by andy5995**
fixes #77

### CODE REVIEW

File: source/packaging-guide/manual.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ Create an AppDir structure that looks (as a minimum) like this::
 	MyApp.AppDir/usr/bin/myapp
 	MyApp.AppDir/usr/lib/libfoo.so.0
 
-The :code:`AppRun` file can be a script or executable. It sets up required environment variables such as :code:`$PATH` and launches the payload application. You can write your own, but in most cases it is easiest (and most error-proof) to use a precompiled one from this repository.
+The :code:`AppRun` file can be a script or executable. It sets up required environment variables such as :code:`$PATH` and launches the payload application. You can write your own, but in most cases it is easiest (and most error-proof) to use a precompiled one from `this repository <https://github.com/AppImage/AppImageBuild>`__.
```
Comment (TheAssassin) [inline source/packaging-guide/manual.rst:30]: That would have been AppImageKit (or, rather, its release page) if anything. However, this section is deprecated. AppImageKit's AppRun often does more harm than good. There should be some words of warning. Something like "... this one might work, but please note its limitations: [list]`.

We should open an issue about the latter point.

**Review by TheAssassin — CHANGES_REQUESTED** [review]

File: source/packaging-guide/manual.rst (hunk 1 above)
Comment (andy5995) [reply source/packaging-guide/manual.rst:30]: Hmmm... well I have no idea what the limitations are.

Comment (phw) [reply source/packaging-guide/manual.rst:30]: @TheAssassin I ran across this issue today as well. Followed the documentation, which advised me to use the AppRun binary, which I then needed to search for.

But if that's not actually what should be used, is there some recommended launch script somewhere?

Comment (TheAssassin) [reply source/packaging-guide/manual.rst:30]: This is not the right place for end user support. Feel free to join our chat: https://docs.appimage.org/contact.html

**Pull request closed without merging**